### DISCUSSION TITLE
Pickup/deliveries segfaults with multiple P&D's chained together.

### DISCUSSION THREAD

**Discussion by stevenshack** [google/or-tools#2747] [Routing (and legacy CP) questions] [answered]
**What version of OR-Tools and what language are you using?**
Version: master/v7.3/v7.2 etc.
Language: C++/Java/Python/C#
Ruby (ankane/or-tools) and C++ backend.

Solver: routing solver

**What operating system (Linux, Windows, ...) and version?**
MacOS

reproduction: Take a basic PD example and modify it with a chained set of pickup and deliveries
```python
# A route a -> b -> c -> d
# The intent here is to take into account labour times at locations.
# So locations B and D are "labour times". Say 20-45 mins each travel time. Physically they're at locations
# A and C.
loca = manager.node_to_index(1)
locb = manager.node_to_index(2)
locc = manager.node_to_index(3)
locd = manager.node_to_index(4)
routing.add_pickup_and_delivery(loca, locb)
routing.add_pickup_and_delivery(locb, locc)
routing.add_pickup_and_delivery(locc, locd)

routing.solver.add(routing.vehicle_var(loca) == routing.vehicle_var(locb))
routing.solver.add(routing.vehicle_var(locb) == routing.vehicle_var(locc))
routing.solver.add(routing.vehicle_var(locc) == routing.vehicle_var(locd))

routing.solver.add(distance_dimension.cumul_var(loca) <= distance_dimension.cumul_var(locb))
routing.solver.add(distance_dimension.cumul_var(locb) <= distance_dimension.cumul_var(locc))
routing.solver.add(distance_dimension.cumul_var(locc) <= distance_dimension.cumul_var(locd))
```

**What did you expect to see**
Chained routes treated as "all or none" pickups and deliveries. 
**What did you see instead?**
Segfaults in ExchangeSubtrip::MakeNeighbor
Unknown which exact line - trace doesn't seem to have symbols internal to methods?

Make sure you include information that can help us debug (full error message, model Proto).

**Anything else we should know about your project / environment**
I'm using ankane/or-tools ruby or-tools wrapper. This should be reproducible in python or c++ examples.
If necessary I will reproduce in Python also.

Planning schedule
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/lib/or_tools/routing_model.rb:14: [BUG] Segmentation fault at 0x0000000000000000
ruby 2.6.6p146 (2020-03-31 revision 67876) [x86_64-darwin19]

-- Crash Report log information --------------------------------------------
   See Crash Report log file under the one of following:                    
     * ~/Library/Logs/DiagnosticReports                                     
     * /Library/Logs/DiagnosticReports                                      
   for more details.                                                        
Don't forget to include the above Crash Report log file in bug reports.     

-- Control frame information -----------------------------------------------
c:0033 p:---- s:0180 e:000179 CFUNC  :solve_with_parameters
c:0032 p:0093 s:0175 e:000174 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/lib/or_tools/routing_model.rb:14
c:0031 p:0059 s:0163 e:000162 METHOD /Users/steven/Code/api/app/mediators/grand_central/dispatcher.rb:798
c:0030 p:0197 s:0159 E:000848 METHOD /Users/steven/Code/api/app/mediators/grand_central/dispatcher.rb:233
c:0029 p:0019 s:0154 E:000870 EVAL   (pry):8 [FINISH]
c:0028 p:---- s:0151 e:000150 CFUNC  :eval
c:0027 p:0044 s:0144 E:0008a0 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:387
c:0026 p:0251 s:0138 E:0008f8 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:355
c:0025 p:0011 s:0127 E:000980 BLOCK  /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:274 [FINISH]
c:0024 p:---- s:0124 e:000123 CFUNC  :catch
c:0023 p:0008 s:0119 E:000960 BLOCK  /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:273 [FINISH]
c:0022 p:---- s:0116 e:000115 CFUNC  :catch
c:0021 p:0022 s:0111 E:000940 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:272
c:0020 p:0062 s:0103 E:0009a8 BLOCK  /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/repl.rb:77 [FINISH]
c:0019 p:---- s:0099 e:000098 CFUNC  :loop
c:0018 p:0005 s:0095 E:002320 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/repl.rb:67
c:0017 p:0005 s:0091 E:002370 BLOCK  /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/repl.rb:38
c:0016 p:0015 s:0088 E:0023c8 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/input_lock.rb:59
c:0015 p:0029 s:0083 E:002430 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/input_lock.rb:77
c:0014 p:0023 s:0077 E:001840 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/repl.rb:38
c:0013 p:0019 s:0073 E:001418 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/repl.rb:13
c:0012 p:0196 s:0068 E:001a38 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_class.rb:200
c:0011 p:0205 s:0061 E:000fe0 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/railties-4.2.11.3/lib/rails/commands/console.rb:110
c:0010 p:0011 s:0057 E:001a68 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/railties-4.2.11.3/lib/rails/commands/console.rb:9
c:0009 p:0087 s:0052 E:0004f8 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/railties-4.2.11.3/lib/rails/commands/commands_tasks.rb:68
c:0008 p:0028 s:0047 E:0026e8 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/railties-4.2.11.3/lib/rails/commands/commands_tasks.rb:39
c:0007 p:0111 s:0042 E:000790 TOP    /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/railties-4.2.11.3/lib/rails/commands.rb:17 [FINISH]
c:0006 p:---- s:0037 e:000036 CFUNC  :require
c:0005 p:0031 s:0032 E:0018c0 TOP    /Users/steven/Code/api/bin/rails:4 [FINISH]
c:0004 p:---- s:0029 e:000028 CFUNC  :debug_load
c:0003 p:0054 s:0020 E:002148 METHOD /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/ruby-debug-ide-2.3.1/lib/ruby-debug-ide.rb:100
c:0002 p:0954 s:0013 E:001d28 EVAL   /Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/ruby-debug-ide-2.3.1/bin/rdebug-ide:204 [FINISH]
c:0001 p:0000 s:0003 E:002690 (none) [FINISH]

-- Ruby level backtrace information ----------------------------------------
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/ruby-debug-ide-2.3.1/bin/rdebug-ide:204:in `<main>'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/ruby-debug-ide-2.3.1/lib/ruby-debug-ide.rb:100:in `debug_program'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/ruby-debug-ide-2.3.1/lib/ruby-debug-ide.rb:100:in `debug_load'
/Users/steven/Code/api/bin/rails:4:in `<top (required)>'
/Users/steven/Code/api/bin/rails:4:in `require'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/railties-4.2.11.3/lib/rails/commands.rb:17:in `<top (required)>'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/railties-4.2.11.3/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/railties-4.2.11.3/lib/rails/commands/commands_tasks.rb:68:in `console'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/railties-4.2.11.3/lib/rails/commands/console.rb:9:in `start'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/railties-4.2.11.3/lib/rails/commands/console.rb:110:in `start'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_class.rb:200:in `start'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/repl.rb:13:in `start'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/repl.rb:38:in `start'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/input_lock.rb:77:in `with_ownership'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/input_lock.rb:59:in `__with_ownership'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/repl.rb:38:in `block in start'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/repl.rb:67:in `repl'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/repl.rb:67:in `loop'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/repl.rb:77:in `block in repl'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:272:in `eval'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:272:in `catch'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:273:in `block in eval'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:273:in `catch'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:274:in `block (2 levels) in eval'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:355:in `handle_line'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:387:in `evaluate_ruby'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pry-0.12.2/lib/pry/pry_instance.rb:387:in `eval'
(pry):8:in `__pry__'
/Users/steven/Code/api/app/mediators/grand_central/dispatcher.rb:233:in `call'
/Users/steven/Code/api/app/mediators/grand_central/dispatcher.rb:798:in `plan_schedule!'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/lib/or_tools/routing_model.rb:14:in `solve'
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/lib/or_tools/routing_model.rb:14:in `solve_with_parameters'

-- Machine register context ------------------------------------------------
 rax: 0x00007f8731507f20 rbx: 0x0000000000000000 rcx: 0x0000000000000000
 rdx: 0x0000000000000006 rdi: 0x00007f874f03b608 rsi: 0x0000000000000001
 rbp: 0x00007ffee3dc46e0 rsp: 0x00007ffee3dc4660  r8: 0x00007f874f03b200
  r9: 0x0000000000000000 r10: 0x0000000000000028 r11: 0x00007f874f9dad80
 r12: 0x00007f874f03b200 r13: 0x00007f874f03b200 r14: 0x00007f874f03b5f0
 r15: 0x0000010000000000 rip: 0x000000010ddb3165 rfl: 0x0000000000010206

-- C level backtrace information -------------------------------------------
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_vm_bugreport+0x82) [0x10c07a472]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_bug_context+0x1d6) [0x10bed4526]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(sigsegv+0x51) [0x10bfe33c1]
/usr/lib/system/libsystem_platform.dylib(_sigtramp+0x1d) [0x7fff71c385fd]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(_ZN19operations_research15ExchangeSubtrip12MakeNeighborEv+0x1a5) [0x10ddb3165]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dcd3140) [0x10dcd3140]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dcd042a) [0x10dcd042a]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dce80c7) [0x10dce80c7]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dce80c7) [0x10dce80c7]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dce55bf) [0x10dce55bf]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dc329e9) [0x10dc329e9]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dc353b0) [0x10dc353b0]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dcf3cf4) [0x10dcf3cf4]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dc33144) [0x10dc33144]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dc31829) [0x10dc31829]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dd493e8) [0x10dd493e8]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/tmp/or-tools/lib/libortools.dylib(0x10dd4802e) [0x10dd4802e]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/or-tools-0.5.3/lib/or_tools/ext.bundle(_ZN4Rice6detail14NativeFunctionIZ12init_routingRNS_6ModuleEE4$_21Lb1EE4callEiPmm+0x337) [0x10d4e99e7]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_call_cfunc+0x149) [0x10c06d4f9]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_exec_core+0x3089) [0x10c054d49]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_vm_exec+0xa98) [0x10c0680f8]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_f_eval+0x2c7) [0x10c062107]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(bind_eval+0x69) [0x10bf8e3e9]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_call_cfunc+0x149) [0x10c06d4f9]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_exec_core+0x3089) [0x10c054d49]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_vm_exec+0xa98) [0x10c0680f8]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(catch_i+0x5e) [0x10c07641e]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_catch_protect+0xd5) [0x10c0637e5]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_f_catch+0x57) [0x10c063ff7]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_call_cfunc+0x149) [0x10c06d4f9]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_exec_core+0x2e3d) [0x10c054afd]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_vm_exec+0xa98) [0x10c0680f8]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(catch_i+0x5e) [0x10c07641e]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_catch_protect+0xd5) [0x10c0637e5]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_f_catch+0x57) [0x10c063ff7]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_call_cfunc+0x149) [0x10c06d4f9]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_exec_core+0x2e3d) [0x10c054afd]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_vm_exec+0xa98) [0x10c0680f8]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(loop_i+0x41) [0x10c0764b1]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_rescue2+0x155) [0x10bedfa85]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_call_cfunc+0x149) [0x10c06d4f9]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_exec_core+0x2e3d) [0x10c054afd]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_vm_exec+0xa98) [0x10c0680f8]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_load_internal0+0x18d) [0x10bf2f42d]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_require_internal+0x30f) [0x10bf2fe7f]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_f_require+0x18) [0x10bf2f558]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_call_cfunc+0x149) [0x10c06d4f9]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_exec_core+0x3089) [0x10c054d49]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_vm_exec+0xa98) [0x10c0680f8]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_load_internal0+0x18d) [0x10bf2f42d]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_load_protect+0x105) [0x10bf2f265]
/Users/steven/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/debase-2.3.2/lib/debase_internals.bundle(zcc639f7e7f+0x1a8) [0x10c320408]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_call_cfunc+0x149) [0x10c06d4f9]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(vm_exec_core+0x3089) [0x10c054d49]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(rb_vm_exec+0xa98) [0x10c0680f8]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(ruby_exec_internal+0xd6) [0x10bedf3c6]
/Users/steven/.rbenv/versions/2.6.6/lib/libruby.2.6.dylib(ruby_run_node+0x49) [0x10bedf259]
/Users/steven/.rbenv/versions/2.6.6/bin/ruby(main+0x5d) [0x10be3bf0d]

[ Several thousand lines of ruby backtrace snipped here]

**Comment by lperron** [comment, marked as answer]
This is a known limitation. You cannot use a node twice (for pickup and for delivery).
You need to duplicate it. Search the mailing list, of these discussions. This was asked and answered multiple times already.

**Comment by Jtoscanini** [comment]
Estimados, que código puedo utilizar en Python y Or Tools para resolver un VRP con dos depósitos?
